supplementary_style_guide/style_guidelines/graphical-interfaces.adoc

```diff
@@ -12,16 +12,6 @@ Bold all graphical user interface elements, including, but not limited to, menu
 Click *OK*.
```

Comment:
@bergerhoffer You might want to kick me, but the same issue applies to this markup as well. The Asciidoc Quick Reference's guidance for GUI buttons is:

Click btn:[Save As] to save the file under a different name.

I suggest using an example of a GUI item that is not a menu or a button, such as a checkbox, or a text field, or an option in a popup menu.

Reply:
Sure we can do that.

Reply:
@stoobie I updated to use a different example. Let me know if that works!